### ISSUE TITLE
Category tiles: sort alphabetically

### ISSUE THREAD

**Issue by so-ale** [so-ale/Bir-United#23]
Sort category tiles alphabetically on the the home page

**Comment by so-ale** [comment]
@xoxobir this will keep you updated whenever this is fixed.

**Comment by snigdharbs** [comment]
Where do I list the service providers? Cabs etc?

**Comment by so-ale** [comment]
Service Providers list can be aggregated on a google docs for now. They can later be put on a separate page on the website.

@ibn-ju and @xoxobir Do you have any other ideas?

**Comment by ibn-ju** [comment]
Yes, let's list them on the Bir United Content Google Doc. We'll find a way to display them.
